### PR TITLE
Change somethings for migration & bug fixes found during migration runs

### PR DESCRIPTION
**Pull request by CarsonF**

### CODE REVIEW

**Review by michaelmarshall — DISMISSED** [review]

File: src/components/partnership/partnership.service.ts [hunk 1]
```diff
@@ -365,32 +358,26 @@ export class PartnershipService {
...
+        changes = {
+          ...changes,
+          financialReportingType: null,
+        };
       }
-      changes = {
-        ...changes,
-        financialReportingType: null,
-      };
+      throw e;
```
Comment (blakekrammes) [inline src/components/partnership/partnership.service.ts:380]: Should you throw an error if managing type is removed and FRT omitted? In that case it's not an invalid operation, just incomplete and you're helping it out above.

Comment (CarsonF) [reply src/components/partnership/partnership.service.ts:380]: Ah! yes this should be in an else case. Good catch.

Comment (blakekrammes) [reply src/components/partnership/partnership.service.ts:380]: Yep. :) Everything else looks solid from what I can tell.

Comment (CarsonF) [reply src/components/partnership/partnership.service.ts]: Sweet! Fixed! 👍

**Pull request merged**